.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+insert_final_newline = false
+max_line_length = 120
```

Comment:
this of course needs to be adjust to your needs as idk the defaults here around i suggested the imho most common.

Reply:
See https://github.com/quarkusio/quarkus/blob/main/independent-projects/ide-config/src/main/resources/eclipse-format.xml for the defaults

Reply:
thx

Reply:
Reconsidering this, IMHO, it only makes sense again with the corresponding `editorconfig-maven-plugin`, as this ensures that the theory (config) is applied in reality (code) as seen here: https://github.com/urlaubsverwaltung/urlaubsverwaltung/blob/5350de17c6ba9b0b1d60e983bb2642b479c5d8bb/pom.xml#L418

Although this is only meant to be an addition to the current Eclipse formatter, as this lacks the `end_of_line = lf` config, or I don’t see it.

Just blocking out the star imports would be compatible and minimally invasive, only causing attention in a really avoidable situation.